src/mango_httpd.erl

```diff
@@ -80,6 +80,14 @@ handle_index_req(#httpd{method='POST', path_parts=[_, _]}=Req, Db) ->
     end,
 	chttpd:send_json(Req, {[{result, Status}, {id, Id}, {name, Name}]});
 
+handle_index_req(#httpd{method='POST', path_parts=[_, <<"_index">>,
+        <<"_bulk_delete">>]}=Req, Db) ->
+    {ok, Opts} = mango_opts:validate_bulk_delete(chttpd:json_body_obj(Req)),
+    DDocs = get_bulk_delete_ddocs_ids(Opts),
```

Comment:
You'll want to rename this DDocIds as well. Sorry to be a nag but I keep reading this as the part that opens the actual #doc{} records and not just grabbing docids from the request body.